### PR TITLE
Added endpoints for projects

### PR DESCRIPTION
**Pull request by Darya-s**
## 📝 Description

- Added endpoint to get all projects from DB
- Added endpoint to count amount of tasks for project with specific id

### CODE REVIEW

File: backend/src/routes/index.ts [hunk 1]
```diff
@@ -9,5 +10,6 @@ const router: Router = express.Router();
 router.use('/auth', auth);
 router.use('/dashboard', validateAuth, dashboard);
 router.use('/user', userRoute);
+router.use('projects', validateAuth, projects);
```
Comment (kralle333) [inline backend/src/routes/index.ts]: looks like there are some inconsistency in how the routes are made and named. I would like you to decide on what you like best and rename the other routes.

File: backend/src/controllers/projects/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  } catch (error) {
+    res.status(StatusCodes.INTERNAL_SERVER_ERROR).send(error);
+  }
+};
+
+export const getAmountOfTasks = async (req: Request, res: Response) => {
+  const { project_id } = req.params;
+  try {
+    const projects = await db('tasks').count('* as tasks_count').where('project_id', project_id);
+    const taskCount = projects[0].tasks_count;
```
Comment (kralle333) [inline backend/src/controllers/projects/index.ts]: Are there always going to be an index 0. You probably want to check for it and handle the situation with an error code.

File: backend/src/routes/projects/index.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import express, { Router } from 'express';
+import { getAllProjects, getAmountOfTasks } from '../../controllers/projects';
+
+const projects: Router = express.Router();
+
+projects.get('/', getAllProjects);
+projects.get('/amount_of_tasks/:project_id', getAmountOfTasks);
```
Comment (kralle333) [inline backend/src/routes/projects/index.ts]: maybe `projects/:project_id/tasks-count` instead, or even `projects/:project_id/tasks/count`

**Review by kralle333 — CHANGES_REQUESTED** [review]
Please check my comments :)

**Review by YurijPaharev — APPROVED** [review]

**Pull request merged**